go/tasks/plugins/array/k8s/config.go

```diff
@@ -34,11 +34,17 @@ var (
 	configSection = config.MustRegisterSection(configSectionKey, defaultConfig)
 )
 
+type ResourceConfig struct {
+	PrimaryLabel string `json:"primaryLabel" pflag:",PrimaryLabel of a given service cluster"`
```

Comment:
This is the cluster name that will be used in resource manager?

Reply:
Yes, that is correct.